requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Base Molo package, pulls in most big things like Wagtail, Django, Pillow
+molo.core == 4.3.2
```

Comment:
So this would be kept up-to-date by requires.io, or whatever and the Docker images will be tagged with this version too (`molo-bootstrap:4.3.2`, `molo-bootstrap:4.3` `molo-bootstrap:4`).

It does maybe mean that you need to keep both the Dockerfile and requirements.txt up-to-date in the project repo if you are pinning the molo.core version in requirements.txt because if the versions of the two differ then the base image is basically useless :-/